baseplate/random.py

```diff
@@ -39,26 +40,26 @@ class WeightedLottery(object):
     """
 
     def __init__(self, items, weight_key):
-        self.tickets = 0
-        self.items_with_weights = []
+        self.weights = []
+        self.items = list(items)
```

Comment:
:no_bell: the 3.6 stdlib does `tuple` here. i wonder if that's significantly better in some way?